### ISSUE TITLE
Failed to activate the atom-overtype-mode package

### ISSUE THREAD

**Issue by steronydh** [brunetton/atom-overtype-mode#21]
[Enter steps to reproduce:]

1. Install
2. Open file
3. Press 'Insert'

**Atom**: 1.19.0-beta2 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [atom-overtype-mode](https://github.com/brunetton/atom-overtype-mode) package 0.3.4

### Stack Trace

Failed to activate the atom-overtype-mode package

```
At editor.getLastCursor(...).onDidChangeVisibility is not a function

TypeError: editor.getLastCursor(...).onDidChangeVisibility is not a function
    at OvertypeMode.prepareEditor (/packages/atom-overtype-mode/lib/atom-overtype-mode.coffee:46:40)
    at /packages/atom-overtype-mode/lib/atom-overtype-mode.coffee:15:8
    at Workspace.observeTextEditors (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/workspace.js:717:15)
    at OvertypeMode.activate (/packages/atom-overtype-mode/lib/atom-overtype-mode.coffee:14:32)
    at Package.module.exports.Package.activateNow (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/package.js:253:25)
    at ~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/package.js:968:31
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/command-registry.js:246:26)
    at ~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/command-registry.js:3:65
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -1:59.2.0 find-and-replace:show (div.panels-item)
  7x -1:55 core:backspace (input.hidden-input)
     -1:50.7.0 editor:consolidate-selections (input.hidden-input)
  2x -1:50.7.0 core:cancel (input.hidden-input)
     -1:44.4.0 core:confirm (input.hidden-input)
     -1:16.8.0 docblockr:parse-enter (input.hidden-input)
     -1:16.8.0 editor:newline (input.hidden-input)
     -1:16.7.0 docblockr:parse-enter (input.hidden-input)
     -1:16.7.0 editor:newline (input.hidden-input)
     -1:16.6.0 core:move-up (input.hidden-input)
     -1:09.8.0 docblockr:parse-enter (input.hidden-input)
     -1:09.8.0 editor:newline (input.hidden-input)
     -1:01.8.0 core:backspace (input.hidden-input)
     -0:55.7.0 core:move-right (input.hidden-input)
     -0:51.7.0 window:save-all (input.hidden-input)
     -0:37.4.0 overtype-mode:toggle (input.hidden-input)
```

### Non-Core Packages

```
atom-alignment 0.13.0 
atom-beautify 0.30.3 
atom-overtype-mode 0.3.4 
autocomplete-clang 0.11.3 
docblockr 0.10.5 
file-icons 2.1.8 
git-time-machine 1.5.9 
highlight-selected 0.13.1 
keybinding-cheatsheet 0.1.1 
minimap 4.28.2 
minimap-bookmarks 0.4.2 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
multi-cursor 2.1.5 
open-recent 5.0.0 
simple-drag-drop-text 0.3.4 
symbols-tree-view 0.14.0 
tab-smart-sort 0.3.0 
todo-show 1.11.0 
```

**Comment by CheeseSucker** [comment]
OnDidChangeVisibility was removed in this commit, and will be gone in atom 1.19:
https://github.com/atom/atom/commit/8aae3ab1ae6bfe79b12cb6f4a26d5073dd4cb2eb

**Comment by mateddy** [comment]
Problem still persist in Atom 1.19.

**Comment by CheeseSucker** [comment]
@mateddy Yes, this error was introduced with Atom 1.19

I'll try to fix this error tonight.

**Comment by CheeseSucker** [comment]
Fixed in atom-overtype-mode@0.4.0
